src/HotChocolate/ElasticSearch/test/Data.ElasticSearch.Tests/__snapshots__/SortTests.ElasticSearch_Sort_Ascending.snap

```diff
@@ -0,0 +1,21 @@
+﻿{
+  "data": {
+    "test": [
+      {
+        "name": "A"
```

Comment:
Does elastic search treat A and a differently?

Reply:
Yeah it does. But it depends on the tokenizer the index is setup with. The default one has this behaviour.